File: MusicWeb.Client/src/models/SongReview.js
export default class SongReview {
  id = null;
  title = "";
  content = "";
  postDate = "";
  songId = "";
  userId = "";
  name = "";
  userName = "";
  albumName = "";
  artistName = "";
  rating = null;
  constructor($data) {
    if ($data) {
      this.id = $data.id;
      this.title = $data.title;
      this.content = $data.content;
      this.postDate = $data.postDate;
      this.songId = $data.songId;
      this.userId = $data.userId;
      this.name = $data.name;
      this.album = $data.album;
      this.artist = $data.artist;
      this.userName = $data.userName;
      this.albumName = $data.albumName;
      this.artistName = $data.artistName;
      this.rating = $data.id;
    }
  }
}

File: MusicWeb.Server/MusicWeb.Models/Entities/Songs/SongReview.cs
using MusicWeb.Models.Entities.Base;
using MusicWeb.Models.Entities.Ratings;
using MusicWeb.Models.Identity;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MusicWeb.Models.Entities
{
    public class SongReview : BaseEntity
    {
        public string Title { get; set; }
        public string Content { get; set; }
        public DateTime PostDate { get; set; }
        public int SongId { get; set; }
        public string UserId { get; set; }
        public virtual Song Song { get; set; }
        public virtual ApplicationUser User { get; set; }
        public virtual SongRating Rating { get; set; }
    }
}
